src/getExtendedPublicKey.c

```diff
@@ -74,18 +74,18 @@ void io_exchange_address()
 {
 	uint32_t tx = 0;
 
-	G_io_apdu_buffer[tx++] = 32;
+	G_io_apdu_buffer[tx++] = PUBLIC_KEY_LENGTH;
 
-	uint8_t rawPublicKey[32];
+	uint8_t rawPublicKey[PUBLIC_KEY_LENGTH];
 	extractRawPublicKey(&data.publicKey, rawPublicKey);
-	os_memmove(G_io_apdu_buffer + tx, rawPublicKey, 32);
-	os_memset(rawPublicKey, 0, 32);
+	os_memmove(G_io_apdu_buffer + tx, rawPublicKey, PUBLIC_KEY_LENGTH);
+	os_memset(rawPublicKey, 0, PUBLIC_KEY_LENGTH);
 
-	tx += 32;
+	tx += PUBLIC_KEY_LENGTH;
```

Comment:
I suggest we rename this to PUBLIC_KEY_SIZE (let's use size for byte sizes and length for array lengths. Note that uint8_t arrays which are basically buffers should have SIZE unless there is a particular concern that type might be changed to uintxy_t)